frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketsTopPanel/statusProperty/statusProperty.styles.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ *  Copyright (C) 2024 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import styled from 'styled-components';
+
+export const Container = styled.div`
+	.MuiChip-root {
+		padding: 0;
+	}
+`;
+
+export const Title = styled.div`
```

Comment:
Do you need to define this here when it is the same as PropertyTitle from issuePropertiesInputs.styles